Formula/benerator.rb

```diff
@@ -0,0 +1,41 @@
+class Benerator < Formula
+  desc "Tool for realistic test data generation"
+  homepage "https://rapiddweller.github.io/homebrew-benerator/"
+  url "https://github.com/rapiddweller/rapiddweller-benerator-ce/releases/download/3.1.0/rapiddweller-benerator-ce-3.1.0-jdk-11-dist.tar.gz"
+  sha256 "194feb051ae18cfcd407b8e1668ce9c60561394bc454f9fc9747c274166843bc"
+
+  license "Apache-2.0"
+  depends_on "openjdk@11"
+  def install
+    # Remove unnecessary files
+    rm_f Dir["bin/*.bat", "bin/pom.xml"]
+
+    # Installs only the 'bin' and 'lib' directories from the tarball
+    # Creates the symlinks to 'bin' scripts in /usr/local/bin
+    # Creates the symlink to the installation path in /usr/local/opt
+    libexec.install Dir["bin", "lib"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+    # Remove non-executable files
+    rm_f "#{bin}/benerator_common"
+    rm_f "#{bin}/log4j2.xml"
+  end
+
+  # opt_prefix is set to the symlink /usr/local/opt/benerator
+  def caveats
+    <<~EOS
+      To use the benerator commands, please set the following environment variables:
+
+      BENERATOR_HOME="#{libexec}"
+      JAVA_HOME="$(/usr/libexec/java_home -v 11)"
```

Comment:
This should be set with a wrapper script like in many Java formula

Reply:
Wrapper script to add the environment variables: 
```ruby  
def install
    # Remove unnecessary files
    rm_f Dir["bin/*.bat", "bin/pom.xml"]

    # Installs only the "bin" and "lib" directories from the tarball
    libexec.install Dir["bin", "lib"]
    # Add wrapper script to set environment variables
    (bin / "benerator").write <<~EOS
      #!/bin/bash
      export JAVA_HOME="$(/usr/libexec/java_home)"
      export BENERATOR_HOME="#{libexec}"
      exec "$BENERATOR_HOME/bin/benerator" "$@"
    EOS
    chmod 0755, bin / "benerator"
    # Remove non-executable files
    rm_f "#{bin}/benerator_common"
    rm_f "#{bin}/log4j2.xml"
  end
```

Removed these paragraph in the EOS and test

Reply:
Please use the default Homebrew wrapper methods for this

Reply:
You can use https://rubydoc.brew.sh/Pathname.html#write_env_script-instance_method to set this

Reply:
I can't really review it in a code block. Please actually push your changes

Reply:
I have pushed the latest update with dd52fdaeaece81f7a93044524c59787804a36001